pkg/apis/openebs.io/v1alpha1/cas_template_keys.go

```diff
@@ -312,3 +329,15 @@ const (
 	// Above templating will give the current repeat resource name
 	CurrentRepeatResourceLITP ListItemsTLPProperty = "currentRepeatResource"
 )
+
+// TaskConfigTLPProperty is used to define properties that comes after TaskConfigTLP
+type TaskConfigTLPProperty string
+
+const (
+	// NameTCTP is a property of TaskConfig this is name of the resource.
+	NameTCTP TaskConfigTLPProperty = "name"
```

Comment:
Let us avoid modifying cas_template_keys completely. Everything should be handled at engine when required.

Reply:
removed